src/lib/containers/entity_finder/details/view/DetailsViewRow.tsx

```diff
@@ -79,24 +77,35 @@ export const DetailsViewRow: React.FunctionComponent<DetailsViewRowProps> = ({
     },
   )
 
+  // TODO: Handle pagination, preferably with an IntersectionObserver
+  // We can't use IntersectionObserver via ref on a div within a <select> input
+  // It might make more sense to use react-select here
+  const { data: versionData } = useGetVersionsInfinite(entityHeader.id, {
```

Comment:
We now have to fetch versions for all versionable entities that come into view. Before we were deferring until the entity was selected. Hopefully shouldn't hurt performance too badly.